lib/mock/mock-agent.js

```diff
@@ -35,12 +40,17 @@ class MockAgent extends Dispatcher {
 
     this[kNetConnect] = true
     this[kIsMockActive] = true
-    this[kMockAgentIsCallHistoryEnabled] = mockOptions?.enableCallHistory ?? false
-    this[kMockAgentAcceptsNonStandardSearchParameters] = mockOptions?.acceptNonStandardSearchParameters ?? false
+    this[kMockAgentIsCallHistoryEnabled] =
+      mockOptions?.enableCallHistory ?? false
+    this[kMockAgentAcceptsNonStandardSearchParameters] =
+      mockOptions?.acceptNonStandardSearchParameters ?? false
+    this[kIgnoreTrailingSlash] = mockOptions?.ignoreTrailingSlash ?? false
 
     // Instantiate Agent and encapsulate
     if (opts?.agent && typeof opts.agent.dispatch !== 'function') {
-      throw new InvalidArgumentError('Argument opts.agent must implement Agent')
+      throw new InvalidArgumentError(
```

Comment:
Agree with @Ethan-Arrowood, can you see if is possible to remove the styling changes so its easier to review the changes for supporting the feature?